showpass/Showpass-E2E-tests
# API login status code verification

**Description:**
Added a few methods to verify API payload and status code.

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-35

**How to test:**
- Run any spec that includes 'Login' and ensure it passes the login portion of it with a proper status code (200)

**Test execution results:**

<img width="1391" alt="Login-api-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/5c561eb6-f1e5-49a9-8055-bf21f56c8f88">
